ncp-web/l10n/de_DE.json

```diff
@@ -0,0 +1,39 @@
+{"translations": {
```

Comment:
I think It would be better to somehow include the ncp-apps name translations in the `l10n/nc-app` subfolder, to keep everything related to any app together